src/wallet/wallet2.h

```diff
@@ -209,6 +209,8 @@ namespace tools
 
       tx_construction_data construction_data;
 
+      bool is_onetime;
```

Comment:
I'm worried about potential inconsistencies in the code outside simplewallet that uses `wallet2::commit_tx(ptx)`, specifically:

src/wallet/api/pending_transaction.cpp 
src/wallet/wallet_rpc_server.cpp 

I've never touched the RPC server etc., so I have no idea what side effects I might have caused.